pan3d/ui/globe.py

```diff
@@ -0,0 +1,883 @@
+import math
+from pathlib import Path
+
+from trame.decorators import TrameApp, change
+from trame.widgets import html, vuetify3 as v3
+
+from pan3d import catalogs as pan3d_catalogs
+from pan3d.utils.constants import XYZ, SLICE_VARS
+from pan3d.utils.convert import max_str_length
+from pan3d.utils.presets import PRESETS
+
+from pan3d.ui.css import base, preview
+from pan3d.ui.collapsible import CollapsableSection
+
+
+class SummaryToolbar(v3.VCard):
```

Comment:
Is that the same as the preview? Can you import and use the one from preview instead?